esrally/track/params.py

```diff
@@ -580,7 +580,7 @@ def percent_completed(self):
         return self.current_bulk / self.total_bulks
 
 
-class ForceMergeParamSouce:
+class ForceMergeParamSouce(ParamSource):
```

Comment:
I noticed there is a typo: It says `ForceMergeParamSouce` but it should be `ForceMergeParamSource` (the "r" is missing in the word "source").

Reply:
🤦‍♂ sorry about that.